docs/releases.md

```diff
@@ -0,0 +1,149 @@
+
+# Release Documentation
+
+# Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+# Basic Terms
+
+## Stablity 
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api).
+- [Internal API](api.md#internal-api) and code compatibility.
```

Comment:
I don't think we have ever offered any stability over that.

Reply:
ACK, removed for now. Probably this hands together with the other about the DSO. Anyway, as long as there's no code exported, no mention needed.

thanks